Instagram/View Controllers/LoginViewController.m

```diff
@@ -0,0 +1,66 @@
+//
+//  LoginViewController.m
+//  Instagram
+//
+//  Created by Sarah Wang on 7/6/21.
+//
+
+#import "LoginViewController.h"
+#import "Parse/Parse.h"
+
+@interface LoginViewController ()
+
+@property (weak, nonatomic) IBOutlet UITextField *usernameField;
+@property (weak, nonatomic) IBOutlet UITextField *passwordField;
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+- (IBAction)registerUser:(id)sender {
+    // initialize a user object
+    PFUser *newUser = [PFUser user];
+    
+    // set user properties
+    newUser.username = self.usernameField.text;
```

Comment:
How do you know there will be text? How could you write this / redesign the user object to avoid a situation where the username is an empty string?